### PR TITLE
Added Calendar WeekView, MonthView

### PR DESCRIPTION
**Pull request by alexanderDracka**
MonthView, WeekView, DaysView, TimezoneSupport, Drag/Resize Validation examples

### CODE REVIEW

**Comment by alexanderDracka** [comment]
Fixed WeekView example, merged upstream changes.

**Comment by alexanderDracka** [comment]
Added Days View and Timezone Support examples. Please @markbrocato look at the Timezone support closely as I am not sure if I chose correct approach in changeOptions method. It is approved way how to change config?

File: packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { Calendar_Day, Panel } from '@extjs/reactor/modern';
+import './data';
+
+export default class CalendarTimezoneSupportExample extends Component{
+    constructor(){
+        super();
+    }
+    changeOptions(){
```
Comment (markbrocato) [inline packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js]: use changeOptions = () => { ... } instead so you don't need to do this.changeOptions.bind(this) later.

File: packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        const selectorValue = Ext.getCmp('selector').getSelection().data.value;
+        this.refs.timezoneview.setConfig('timezoneOffset', selectorValue)
```
Comment (markbrocato) [inline packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js]: This should be replaced with this.setState({ timezoneOffset: selectorValue }) instead.

File: packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                    value: -60
+                }, {
+                    text: Ext.os.is.Phone ? 'Sydney +10' : 'Sydney (UTC+10:00)',
+                    value: -600,
+                }]
+            }]
+        }
+    }
+
+    calendarTimezoneConfig={
```
Comment (markbrocato) [inline packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js]: This should go in the render method as the props for <Calendar_Day>, timezoneOffset should be set to {this.state.timezoneOffset}.

File: packages/reactor-kitchensink/src/examples/Calendar/WeekView.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import { Calendar_Week, Panel } from '@extjs/reactor/modern';
+import './data';
+
+export default class CalendarWeekViewExample extends Component{
+    constructor(){
+        super();
+    }
+
+    changeToWorkWeek(){
```
Comment (markbrocato) [inline packages/reactor-kitchensink/src/examples/Calendar/WeekView.js]: Use ES2017 property initializer changeToWorkWeek = () => {...}

**Comment by alexanderDracka** [comment]
@markbrocato sorry for these commits as I should made separate PR. Will do next time.
Added Fix for EdgeMenu and removed one deprecated config in Basic markers

**Pull request merged**